Add note to span examples not representing a particular JSON format (#4311)
open-telemetry/opentelemetry.io: content/en/docs/concepts/signals/traces.md

Let's explore this with three units of work, represented as [Spans](#spans):

> > 📝 **Note**: The JSON examples below do not represent a particular format, 
> > and especially not OTLP JSON which is much more verbose.
> 
> 

Reviewer: ```suggestion
{{% alert title="Note" %}}

The JSON examples below do not represent a particular format, 
and especially not OTLP JSON which is much more verbose.

{{% /alert %}}
```

Although the `>` renders as a note as well using the alert shortcode is the preferred way, because it gives us some flexibililty in color, title, etc.

Author: Will do. What is the preferred link for adding here, please?

Let's explore this with three units of work, represented as [Spans](#spans):

{{% alert title="Note" %}}

> The JSON examples below do not represent a particular format, and especially not
> [OTLP/JSON](/docs/specs/otlp/#json-protobuf-encoding) which is much more
> verbose.

Reviewer 2: ```suggestion
The following JSON examples do not represent a specific format, and especially not
[OTLP/JSON](/docs/specs/otlp/#json-protobuf-encoding), which is more verbose.
```

Let's explore this with three units of work, represented as [Spans](#spans):

{{% alert title="Note" %}}

The JSON examples below do not represent a particular format, and especially not
[OTLP/JSON](/docs/specs/otlp/#json-protobuf-encoding) which is much more
verbose.

> {{% /alert %}} `hello` span:

Reviewer: that looks odd, I guess an auto formatter issue?

Reviewer: ```suggestion
{{% /alert %}}

`hello` span:
```
